firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -822,7 +856,13 @@ private void doCloseSessions(int maxCustomExceptionEvents, boolean excludeCurren
     // maximum chance that the user code that sets this information has been run.
     writeSessionUser(mostRecentSessionIdToClose);
 
+    if (!excludeCurrent) {
```

Comment:
Is it worth flipping the boolean to be `includeCurrent` to avoid the double negative?

Reply:
Yeah, was trying to minimize the amount of change in this one PR, but I'll look at it.